src/app/components/DataDetailPage.tsx

```diff
@@ -0,0 +1,237 @@
+'use client';
+
+import { Fragment } from 'react';
+import { Box, Flex, Text, Icon, Link, Image } from '@chakra-ui/react';
+import { MdLocationOn } from 'react-icons/md';
+import { RiBuildingFill } from 'react-icons/ri';
+import { AiOutlineDollar, AiOutlineClose } from 'react-icons/ai';
+import { TbBrandGoogleAnalytics } from 'react-icons/tb';
+import styled from 'styled-components';
+
+const Btn = styled.button`
+  width: 77%;
+  height: 44px;
+  text-align: center;
+  color: #fff;
+  background-color: #00ba61;
+  border: 0;
+  border-radius: 0.5rem;
+  font-size: 12px;
+  font-weight: 700;
+  margin-top: 100px;
+  margin-bottom: 20px;
+`;
+
+type ChildComponentProps = {
+  key: number;
+  Title: string;
+  Company: string;
+  Description: string;
+  Location: string;
+  Salary: number;
+  Level: string;
+  Type: string;
+  Function: string;
+  Industry: string;
+  URL: string;
+  image_url: string;
+  handlePopUpState: () => void;
+};
+
+export default function DataDetailPage({
```

Comment:
The text and components should be centered among the pop space and the space given to them should be wider.
<img width="868" alt="image" src="https://user-images.githubusercontent.com/5522386/230512439-0f8dfcc8-6e05-46b7-b458-ca3632a4b273.png">

Reply:
In addition, The `Entry Level` with the icon should start at 50% of the content or just split to the ends if that is easier.

Reply:
I am using an icon and have not found a way to fill half of it.